pipeline.py

```diff
@@ -9,7 +9,6 @@
     CreatePVC,
     DeletePVC,
     mount_pvc,
-    set_image_pull_policy,
```

Comment:
This is actually used here:
https://github.com/opendatahub-io/ilab-on-ocp/blob/ac474e1cfbb64a1da76502c1f65199508b78a3dc/pipeline.py#L420-L421

So I would not remove it. But probably we want to implement a better solution than having the user uncomment code.

Reply:
we used this for test/development - we can add a var to make the imagePullPolicy configurable, or remove it and  manually add it whenever we need it, or keep it commented & as/is for convenience so we don't have to remember how to set it in the future - removing it might make sense now that we are using the upstream image & any update to that will come with a new tag.

Reply:
Re-added, thanks!